### PR TITLE
when detect Indentation is false set the editior.tabSize based on settings value

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
### What does this PR do?

https://user-images.githubusercontent.com/93245779/147567212-77383a76-bdfb-4701-b4fd-3bf9fbf0b99c.mp4

### What issues does this PR fix or reference?

https://github.com/redhat-developer/vscode-yaml/issues/573

### Is it tested? How?
Yes. with test case

### CODE REVIEW

File: src/languageserver/handlers/settingsHandlers.ts [hunk 1]
```diff
@@ -146,6 +153,17 @@ export class SettingsHandler {
     }
   }
 
+  public async updateTabSize(tabSize: number): Promise<boolean> {
```
Comment (evidolob) [inline src/languageserver/handlers/settingsHandlers.ts:156]: Not sure, why we need to update user settings?

Comment (msivasubramaniaan) [reply src/languageserver/handlers/settingsHandlers.ts:156]: As per the bug, when user changed the settings Editor: Detect Indentation is false, then we need to change the default tab size to Editor: Tab size settings value. For that we need to update the user settings from LS.

Comment (evidolob) [reply src/languageserver/handlers/settingsHandlers.ts:156]: I still not understand why we need to update property. If Editor: Detect Indentation is false formatter should use Editor: Tab, but we don't need to change any property for that

Comment (msivasubramaniaan) [reply src/languageserver/handlers/settingsHandlers.ts]: Removed the update property code.

**Review by evidolob — APPROVED** [review]

**Pull request merged**